native/kotlin/api/kotlin/src/integrationTest/kotlin/IntegrationTestHelpers.kt

```diff
@@ -8,7 +8,7 @@ const val SECOND_USER_ID: UserId = 2
 const val FIRST_USER_EMAIL = "test@example.com"
 const val SECOND_USER_EMAIL = "themeshaperwp+demos@gmail.com"
 const val SECOND_USER_SLUG = "themedemos"
-const val NUMBER_OF_USERS = 3
+const val NUMBER_OF_USERS = 4
```

Comment:
Weird nit – I wonder if this should be `EXPECTED_ NUMBER_OF_USERS`?

Reply:
That makes sense, although I want to move all of these into the `test_credentials.json` file and in that file number of users will make more sense.